From e4ada42a4b1edf7bc381405eb040000ca3e4c222 Mon Sep 17 00:00:00 2001
From: Vincent ABRIOU <vincent.abriou@st.com>
Date: Tue, 23 Mar 2021 15:40:34 +0100
Subject: [PATCH 1/1] fix noexcept warning issue with Yocto cross compile.

BorderSize(unsigned int size) and PixelValue() must be declared as noexcept
to avoid cross compilation warning with [-Werror=noexcept]

Signed-off-by: Vincent ABRIOU <vincent.abriou@st.com>
---
 arm_compute/core/PixelValue.h | 2 +-
 arm_compute/core/Types.h      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arm_compute/core/PixelValue.h b/arm_compute/core/PixelValue.h
index f744d6443..53b400f35 100644
--- a/arm_compute/core/PixelValue.h
+++ b/arm_compute/core/PixelValue.h
@@ -35,7 +35,7 @@ class PixelValue
 {
 public:
     /** Default constructor: value initialized to 0 */
-    PixelValue()
+    PixelValue() noexcept
         : value{ int64_t(0) }
     {
     }
diff --git a/arm_compute/core/Types.h b/arm_compute/core/Types.h
index 59ba5aabb..d6a46bbac 100644
--- a/arm_compute/core/Types.h
+++ b/arm_compute/core/Types.h
@@ -279,7 +279,7 @@ struct BorderSize
     }
 
     /** Border with equal size around the 2D plane */
-    explicit constexpr BorderSize(unsigned int size)
+    explicit constexpr BorderSize(unsigned int size) noexcept
         : top{ size }, right{ size }, bottom{ size }, left{ size }
     {
     }
-- 
2.17.1

